### PR TITLE
u-boot-envtools: update to 2020.04

### PR DESCRIPTION
**Pull request by lucize**
also change the download link to the new site

Signed-off-by: Lucian Cristian <lucian.cristian@gmail.com>

tested on mvebu,rockchip

### CODE REVIEW

File: package/boot/uboot-envtools/Makefile [hunk 1]
```diff
@@ -9,16 +9,16 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=uboot-envtools
 PKG_DISTNAME:=u-boot
-PKG_VERSION:=2019.07
-PKG_RELEASE:=3
+PKG_VERSION:=2020.04
+PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE:=$(PKG_DISTNAME)-$(PKG_VERSION).tar.xz
```
Comment (neheb) [inline package/boot/uboot-envtools/Makefile:16]: Not needed.

File: package/boot/uboot-envtools/Makefile [hunk 2]
```diff
@@ -9,16 +9,16 @@ include $(TOPDIR)/rules.mk
...
 PKG_SOURCE_SUBDIR:=$(PKG_DISTNAME)-$(PKG_VERSION)
```
Comment (neheb) [inline package/boot/uboot-envtools/Makefile:17]: Not needed.

File: package/boot/uboot-envtools/Makefile [hunk 3]
```diff
@@ -9,16 +9,16 @@ include $(TOPDIR)/rules.mk
...
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_DISTNAME)-$(PKG_VERSION)
```
Comment (neheb) [inline package/boot/uboot-envtools/Makefile:18]: I want to say not needed.

**Comment by lucize** [comment]
I'll use https://ftp.denx.de/pub/u-boot/ as u-boot.mk is already downloading a bz2 from ftp:// and correct there also

File: package/boot/uboot-envtools/Makefile (hunk 1 above)
Comment (lucize) [reply package/boot/uboot-envtools/Makefile]: needed

File: package/boot/uboot-envtools/Makefile (hunk 2 above)
Comment (lucize) [reply package/boot/uboot-envtools/Makefile:18]: needed

File: package/boot/uboot-envtools/Makefile (hunk 3 above)
Comment (lucize) [reply package/boot/uboot-envtools/Makefile:19]: needed

**Comment by hauke** [comment]
Thank you for your patch, it was applied to master.

**Pull request closed without merging**